doc/rados/operations/crush-map-edits.rst

```diff
@@ -32,193 +33,199 @@ Pool Values`_.
 
 .. _getcrushmap:
 
-Get a CRUSH Map
----------------
+Get the CRUSH Map
+-----------------
 
-To get the CRUSH map for your cluster, execute the following:
+To get the CRUSH map for your cluster, run a command of the following form:
 
 .. prompt:: bash $
 
-	ceph osd getcrushmap -o {compiled-crushmap-filename}
+    ceph osd getcrushmap -o {compiled-crushmap-filename}
 
-Ceph will output (-o) a compiled CRUSH map to the filename you specified. Since
-the CRUSH map is in a compiled form, you must decompile it first before you can
-edit it.
+Ceph outputs (``-o``) a compiled CRUSH map to the filename that you have
+specified. Because the CRUSH map is in a compiled form, you must first
+decompile it before you can edit it.
 
 .. _decompilecrushmap:
 
-Decompile a CRUSH Map
----------------------
+Decompile the CRUSH Map
+-----------------------
 
-To decompile a CRUSH map, execute the following:
+To decompile the CRUSH map, run a command of the following form:
 
 .. prompt:: bash $
 
-	crushtool -d {compiled-crushmap-filename} -o {decompiled-crushmap-filename}
+    crushtool -d {compiled-crushmap-filename} -o {decompiled-crushmap-filename}
 
 .. _compilecrushmap:
 
-Recompile a CRUSH Map
----------------------
+Recompile the CRUSH Map
+-----------------------
 
-To compile a CRUSH map, execute the following:
+To compile the CRUSH map, run a command of the following form:
 
 .. prompt:: bash $
 
-	crushtool -c {decompiled-crushmap-filename} -o {compiled-crushmap-filename}
+    crushtool -c {decompiled-crushmap-filename} -o {compiled-crushmap-filename}
 
 .. _setcrushmap:
 
 Set the CRUSH Map
 -----------------
 
-To set the CRUSH map for your cluster, execute the following:
+To set the CRUSH map for your cluster, run a command of the following form:
 
 .. prompt:: bash $
 
-	ceph osd setcrushmap -i {compiled-crushmap-filename}
+    ceph osd setcrushmap -i {compiled-crushmap-filename}
```

Comment:
Tempted to prefix this with _pucker tight_ ;)  Kids today don't know how good they have it ;)

Reply:
As tempting as it is to instruct our readers in volitional control of their alimentary canals, I ultimately have to recognize the reality that we are running an upstream technical writing department, not a colorectal charm school.